Soporte para GNU/Linux?

Navegando por el repo note que no solo en la apariencia de la aplicación hay un grán parecido a popcorn time app, si no que también el código tiene una buena parte extraida de esta otra aplicación. Veo que no sería muy complicado añadir las instrucciones de X

Gracias @ivantodorovich hice lo que mencionas, pero al darle npm install en un momento tira este error:
npm ERR! nodewebkit@0.9.2 postinstall: `node scripts/install.js`
npm ERR! Exit status 8
npm ERR! 
npm ERR! Failed at the nodewebkit@0.9.2 postinstall script.
npm ERR! This is most likely a problem with the nodewebkit package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node scripts/install.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls nodewebkit
npm ERR! There is likely additional logging output above.

npm ERR! System Linux 3.13-1-amd64
npm ERR! command "/usr/bin/nodejs" "/usr/bin/npm" "install"
npm ERR! cwd /home/usuario/Descargas/Cuevana Storm/storm
npm ERR! node -v v0.10.26
npm ERR! npm -v 1.4.4
npm ERR! code ELIFECYCLE

Luego continua ejecutando todo correctamente y al final
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /home/usuario/Descargas/Cuevana Storm/storm/npm-debug.log
npm ERR! not ok code 0

al darle npm start, corre la aplicación, pero al cargar una pelicula sigue quedando negra la pantalla. Tengo el npm-debug.log, pero no se donde subirlo porque es larguisimo.
Gracias!

--- a/app/js/main.js
+++ b/app/js/main.js
@@ -154,7 +154,7 @@ var Storm = function() {
 		} else if (isMac) {
 			$('body').addClass('isMac');
 		} else if (isLinux) {
-			$('body').addclass('isLinux');
+			$('body').addClass('isLinux');
 		}
 
 		// Window height containers

